### DISCUSSION TITLE
asas aladas

### DISCUSSION THREAD

**Discussion by rafaelmacedo2890** [thatskymod/Sky-CotL-Scripts#80] [Q&A]
como faço para tirar as asas que eu peguei?
já fui ao eden perde mas elas sempre voltam

**Comment by thatskymod** [comment]
Quando ascenderes através do Olho do Éden, é importante reconheceres que irás recuperar um certo número de Bônus de Asa - até 49 de Espíritos Regulares e até 64 de Espíritos Viajantes. Isto significa que podes ter um avanço de até 113 Bônus de Asa para o teu próximo nascimento. Embora isto não esteja necessariamente relacionado com os mods, é importante ter isto em mente.

Se te encontras com um excesso de asas, aqui estão algumas opções que podes tentar:

1. Sacrifica-te aos [Dragões Sombrios](https://sky-filhos-da-luz.fandom.com/pt-br/wiki/Drag%C3%B5es_Sombrios).
2. Procurar o exato oposto da Coleção de Asas: Em [AW Dukun](https://github.com/thatskymod/Sky-CotL-Scripts/blob/d4dbf26e5072572cd073291270446e5616fa49f7/Canvas/Libs/libdukun151.so) e [Langit](https://github.com/thatskymod/Sky-CotL-Scripts/blob/d4dbf26e5072572cd073291270446e5616fa49f7/Canvas/Libs/libLangit023rc.so), tens a opção de "Largar" as tuas asas. Não te esqueças de reiniciar o jogo depois para veres os efeitos.
3. Costumava haver uma opção para explodir as asas no LibLangit e em vários outros scripts, mas parece não estar disponível neste momento.

Avisa-me se isto ajudar!

<details>
<summary>English Version (click to expand)</summary>
When you ascend through the Eye of Eden, it's important to recognize that you will regain a certain number of Wing Buffs—up to 49 from Regular Spirits and up to 64 from Traveling Spirits. This means you could have a head start of up to 113 Wing Buffs for your next birth. Although this is not necessarily related to mods, it's valuable to keep in mind.

If you find yourself with an excess of wings, here are a few options you can try:

1. Sacrifice yourself to [Dark Dragons](https://sky-children-of-the-light.fandom.com/wiki/Dark_Dragons).
2. Pursue the exact opposite of Wing Collection: In [AW Dukun](https://github.com/thatskymod/Sky-CotL-Scripts/blob/d4dbf26e5072572cd073291270446e5616fa49f7/Canvas/Libs/libdukun151.so) and [Langit](https://github.com/thatskymod/Sky-CotL-Scripts/blob/d4dbf26e5072572cd073291270446e5616fa49f7/Canvas/Libs/libLangit023rc.so), you have the option to "Drop" your wings. Make sure to restart the game afterward to see the effects.
3. There used to be an option to explode wings in LibLangit and several other scripts, but it appears to be unavailable at the moment.

Let me know if this helps!
</details>

**Comment by rafaelmacedo2890** [comment]
Tem como eu tirar esses bônus que eu não tinha antes, mas fiz algo no mod e
acabou ficando preso com essas asas eu ficava só com 36 por conta de
espíritos agora estou ficando com 120 sempre que morro
vocês poderiam me ajudar?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Não
contém vírus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by thatskymod** [comment]
Obrigado pela vossa paciência enquanto eu esclarecia os pormenores. Infelizmente, a minha investigação mostrou que é impossível apagar os bónus das asas dos espíritos. Uma vez recolhidos, ficam permanentemente armazenados na tua conta sem possibilidade de remoção. Isto pode fazer com que a tua conta corra um risco acrescido de ser banida. Por exemplo, o libsmite avisa-o antes de executar esta ação, como mostra a imagem abaixo.

<details>
<summary>English Version (click to expand)</summary>
Thank you for your patience while I clarified the details. Unfortunately, my research has shown that deleting the wing buffs of spirits is impossible. Once collected, they are permanently stored on your account without the possibility of removal. This may put your account at an increased risk of being banned. For instance, libsmite warns you before performing this action, as shown in the picture below.
</details>

![1000588639](https://github.com/thatskymod/Sky-CotL-Scripts/assets/90126520/ee59264c-9bc2-4b11-80fe-f00c05b09b2b)

**Reply by 2021em121-hue** [reply]
Tôi muốn bản mod này quay trở lại thêm lần nữa

**Reply by thatskymod** [reply]
sMite is not coming back. It has been abandoned/discontinued by the author. Use libTSM instead.

**Comment by thatskymod** [comment]
This question appears to have been answered.

Due to prolonged inactivity on this topic, it seems appropriate to close.

If you have any further questions, feel free to reach out!
